src/runtime_src/core/pcie/tools/xbmgmt/xqspips.cpp

```diff
@@ -35,6 +35,7 @@
 
 #define SAVE_FILE                   0
 #define FLASH_BASE                  0x040000
+#define GOLDEN_BASE                 0x06000000
```

Comment:
The golden start from 96MB. But does this the case on U30?

Reply:
u30 will follow the same to u25